packages/cli/src/commands/admin-web/local-dev.ts

```diff
@@ -22,9 +22,9 @@ console.log('CLI spoolin');
 
 async function main() {
   /* eslint-disable-next-line no-restricted-syntax */
-  const targetDir = path.resolve(__dirname, 'embedded/');
```

Comment:
If we want to have the `admin-web` app in its own package (which is a good idea!), let's properly have `@backstage/cli` depend on it and import it accordingly.

Additionally, I'm guessing that you'll want to move the "dev the admin page` yarn script from `@backstage/cli` to here (I think you should move this `local-dev.ts` file and friends).

Reply:
I tried to move it but I'm getting an error for using the `path` library in a frontend package. So I'm thinking of either keeping it where it is or creating another package (backend). WDYT?

Reply:
Ah, weird, in that case I'd recommend leaving it as-is, and I'll take a look as part of making `@backstage/cli` build with the admin web app embedded :+1:

Reply:
The point is the react package is a frontend package and the `local-dev` is backend. so they can be together in a package.

Reply:
I don't fully understand your comment here.

Currently, `local-dev` has no backend-related behaviour, and its only purpose is to launch a "dev build" of the frontend, which is arguably a frontend-related task.

Either way, I don't think that this should block your PR at this point :+1:

Reply:
`local-dev` is a `node.js` code and when I'm moving the file to my frontend package (admin-web)`yarn tsc` raises an error `error  'path' import is restricted from being used  `.

Reply:
> Ah, weird, in that case I'd recommend leaving it as-is

😄 Right, I'd recommend leaving the file as you had it, then.
I'll see if there's any workarounds when I work on the `@backstage/cli` "production" build.